tycho-bundles/org.eclipse.tycho.embedder.shared/src/main/java/org/eclipse/tycho/TychoConstants.java

```diff
@@ -64,6 +64,8 @@ public interface TychoConstants {
 
     String PROP_PGP_SIGNATURES = "pgp.signatures";
 
+    String PROP_WRAPPED_ORIGINAL_GAV = "wrapped.originalGav";
```

Comment:
we already have some "maven-groupId etc. properties (see above), so I think we should use the maven prefix here as well, maybe `maven-gav-hint`, or even just duplicate all of them using

- `maven-wrapped-groupId`
- `maven-wrapped-artifactId`
- ...

That way it is much more flexible /extensible and probably one can even reuse the existing Properties advice and just pass the prefix (`maven-` or `maven-wrapped-`)

Reply:
Splitting the constants into the prefix (`maven-` and `maven-wrapped`) and property (`groupId`, ...) makes them look very messy.
I think it's cleaner to simply pass a map of properties. This way I can still reuse the existing class.